packages/components/src/components/files/FileTree.tsx

```diff
@@ -182,6 +185,7 @@ export class FileTree extends PureComponent<FileTreeProps, FileTreeState> {
     static defaultProps = {
         allowMultiSelect: true,
         useFileIconCls: false,
+        mvDirMode: false,
```

Comment:
per suggestion above:
```suggestion
        emptyDirectoryText: 'No Files Found',
```